template/src/store/selectors/app.ts

```diff
@@ -1,6 +1,6 @@
-import {IApp} from '../../constants/interface/redux/AppInterface'
+import { IApp, IInitialState } from '../types'
 
-const getAppData = (state): IApp => state.app
+const getAppData = (state: IInitialState): IApp => state.app
 
-export const getLoadingIndicator = state => getAppData(state).showGlobalIndicator
-export const getAppStackState = (state): string => getAppData(state).appState
+export const getLoadingIndicator = (state: IInitialState): boolean => getAppData(state).showGlobalIndicator ?? false
```

Comment:
we need nullish coalescing for showGlobalIndicator ? 
I see you already limit the boolean value when setShowGlobalIndicator

Reply:
resolved